tests/system/data_sources/test_postgres.py

```diff
@@ -662,6 +681,32 @@ def test_column_validation_core_types_to_bigquery():
     )
 
 
+@mock.patch(
+    "data_validation.state_manager.StateManager.get_connection_config",
+    new=mock_get_connection_config,
+)
+def test_column_validation_view_core_types_vw():
+    """PostgreSQL to PostgreSQL view dvt_core_types_vw column validation"""
+    # TODO Change --sum and --max options to include col_char_2 when issue-842 is complete.
+    cols = ",".join(
+        [
+            _
+            for _ in DVT_CORE_TYPES_COLUMNS
+            if _ not in ("id", "col_float32", "col_char_2")
```

Comment:
```suggestion
            if _ not in ("id",  "col_char_2")
```
col_float32 doesn't need to be removed for PG to PG validation